### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by g-marcin**
Shop-cart first PR from to main branch.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,130 @@
...
+      >
+        <i class="fa-solid fa-bars fa-xl"></i>
+      </button>
+    </header>
+    <main>
+      <div class="wrapper__Main">
+        <div class="wrapper__Shop">
+          <div class="header__Shop"><i class="fa-solid fa-shop"></i> THE SHOP</div>
+          <div class="content__Shop">
+            <div class="wrapper__Product" style="display: none">
```
Comment (jjuszkiewicz) [inline index.html:57]: a good practice is rid of style=...

File: index.html [hunk 2]
```diff
@@ -7,6 +7,130 @@
...
+        </div>
+        <div class="separator" style="min-width: 5vw"></div>
+        <button
+          class="dropdown__Account"
+          aria-label="open account menu"
+          type="menu"
+          style="margin: 0; padding: 0"
+          onclick="test(e)"
+        >
+          <a href="#" title="Account menu" aria-label="account dropdown menu">Account </a>
```
Comment (jjuszkiewicz) [inline index.html:38]: `<a>`inside button the same mistake as in HTML task 😉

File: index.js [hunk 3]
```diff
@@ -0,0 +1,487 @@
+let cartProducts = [];
+let brandSet = new Set();
```
Comment (jjuszkiewicz) [inline index.js:2]: The approach with so many state will cause lot of problems. Try to have only cart and products state.

**Review by jjuszkiewicz — COMMENTED** [review]
Your solution is OK. However, too many states make the code too complicated. Let's talk about it on the meeting

File: index.html (hunk 1 above)
Comment (g-marcin) [reply index.html:57]: Well noted. I will remove inline styles and avoid using it in the future.

File: index.html (hunk 2 above)
Comment (g-marcin) [reply index.html:38]: Well noted.

File: index.js (hunk 3 above)
Comment (g-marcin) [reply index.js:2]: Well noted. At present global state variables reduced to cartProducts array and fetchedProducts map . Going to remove fetchedProductMap state.

**Pull request merged**